### DISCUSSION TITLE
is there a survival guide or first steps manual on actions one should do when you got a crashed cluster? (standalone in my case)  How to repair not working milvus?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#34572] [Q&A and General discussion]
Is there a survival guide or first steps manual on actions one should do when you got a crashed cluster? (standalone in my case) 
Like what should you do to try repair broken cluster. And what actions are most efficient?

Because a quite often stubbled upon this situation, especially after sudden reboot.

**Comment by yhmo** [comment]
Usually, sudden reboot is caused by OOM, you can increase the memory capacity of the container.
If the milvus container fails to restart, you can export the log by command `docker logs [container id] > 1.log`. There might be errors or warnings in the log. Search the error string in the [milvus issues](https://github.com/milvus-io/milvus/issues) sometimes you can find similar issues and add your comment. If there is no similar issue there, you can raise a new issue and attach your log to the issue.
